Core/WebViewController.swift

```diff
@@ -190,11 +190,8 @@ open class WebViewController: UIViewController, WKNavigationDelegate, WKUIDelega
     }
     
     fileprivate func touchesYOffset() -> CGFloat {
```

Comment:
To allow it to be used by an internal private extension (which falls outside of class scope).